tern/command_lib/base.yml

```diff
@@ -96,6 +97,13 @@ dpkg:
           - "pkgs=`dpkg --get-selections | cut -f1 -d':' | awk '{print $1}'`"
           - "for p in $pkgs; do cat /usr/share/doc/$p/copyright; echo LICF; done"
     delimiter: 'LICF'
+  files:
+    invoke:
+      1:
+        container:
+        - "pkgs=`dpkg --get-selections | cut -f1 -d':' | awk '{print $1}'`"
+        - 'for p in $pkgs; do files=`dpkg-query -L $p`; for file in $files; do if [[ -f $file ]]; then checksum=`sha256sum $file`; attrs=`ls -l $file`; echo "$checksum|$attrs"; fi; done; echo LICF; done'
```

Comment:
What would be cool here is if you could only collect file paths, and then look in the `files` property for the layer to see if a FileData object for that layer exists and copy it in here. That way you don't have to do the double work of calculating checksums and whatnot. On the other hand, you will have to figure out how to just get a list of file paths.

Reply:
That is a nice efficient idea. I will see what I can do.